projects/admin/src/app/classes/library.ts

```diff
@@ -19,9 +19,14 @@
 // required as json properties is not lowerCamelCase
 
 import { WeekDay } from '@angular/common';
+import { marker } from '@biesbjerg/ngx-translate-extract-marker';
```

Comment:
`import { marker as _ } from '@biesbjerg/ngx-translate-extract-marker';`

Reply:
Unfortunately this does not works with the tests.